cmd/run.go

```diff
@@ -164,6 +164,7 @@ To skip bundle verification, use the --skip-verify flag.
 
 	addConfigFileFlag(runCommand.Flags(), &cmdParams.rt.ConfigFile)
 	runCommand.Flags().BoolVarP(&cmdParams.serverMode, "server", "s", false, "start the runtime in server mode")
+	runCommand.Flags().DurationVarP(&cmdParams.rt.WaitBundleTimeout, "wait-bundle-timeout", "", 0, "wait for configured bundles activation before starting server")
```

Comment:
nit: Lets rename the flag to `--ready-timeout` or similar. One lesson learned with the `/health` API for ready checks is that what people will likely want (eventually) is the equivalent of `/health?bundles&plugins`. Where the server should wait for all configured bundles _and_ plugins to be initialized.

We should also specify in the help text that any value <=0 means the ready check is disabled.